app/javascript/gobierto_data/webapp/components/sets/SQLEditorTable.vue

```diff
@@ -40,27 +49,50 @@ export default {
       default: () => {
         return []
       }
+    },
+    numberRows: {
+      type: Number,
+      required: true
+    },
+    tableName: {
+      type: String,
+      required: true
     }
   },
   data() {
     return {
       keysData: [],
-      mutableList: this.items
+      mutableList: this.items,
+      showTotalRows: false,
+      labelButtonRows: ''
     }
   },
   watch:{
     items: function(){
-        this.mutableList = JSON.parse(this.items);
+      this.mutableList = JSON.parse(this.items);
```

Comment:
watch directive receives two params: `newValue, oldValue`. this.items might haven't been updated properly, so better `this.mutableList = JSON.parse(newVal);`